db/db_impl/db_impl_write.cc

```diff
@@ -1771,9 +1779,13 @@ Status DBImpl::SwitchMemtable(ColumnFamilyData* cfd, WriteContext* context) {
     // We may have lost data from the WritableFileBuffer in-memory buffer for
     // the current log, so treat it as a fatal error and set bg_error
     if (!io_s.ok()) {
-      error_handler_.SetBGError(io_s, BackgroundErrorReason::kMemTable);
+      // Should handle return error?
```

Comment:
I mentioned this in another PR but can we create a signature of SetBGError that does not return a Status?  If everyone ignores it, why return it?